### PR TITLE
Migrate command_help to null safety

### PR DESCRIPTION
**Pull request by jmagman**
Convert `CommandHelpOption` getters to `late final`s.

Part of #71511

### CODE REVIEW

File: packages/flutter_tools/lib/src/base/command_help.dart [hunk 1]
```diff
@@ -38,158 +34,134 @@ class CommandHelp {
...
-  CommandHelpOption _k;
-  CommandHelpOption get k => _k ??= _makeOption(
+  late final CommandHelpOption k = _makeOption(
     'k',
     'Toggle CanvasKit rendering.',
   );
 
-  CommandHelpOption _M;
-  CommandHelpOption get M => _M ??= _makeOption(
+  late final CommandHelpOption M = _makeOption(
```
Comment (jonahwilliams) [inline packages/flutter_tools/lib/src/base/command_help.dart:164]: Seems like a reasonable use of `late final`

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/base/command_help.dart (hunk 1 above)
Comment (jmagman) [reply packages/flutter_tools/lib/src/base/command_help.dart:164]: Levelling up!

**Comment by jmagman** [comment]
Google testing is mad it's not on https://github.com/flutter/flutter/pull/78816, merging.

**Pull request merged**